app/src/main/java/com/zeapo/pwdstore/git/BaseGitActivity.kt

```diff
@@ -33,7 +34,7 @@ import net.schmizz.sshj.userauth.UserAuthException
  * Abstract [AppCompatActivity] that holds some information that is commonly shared across git-related
  * tasks and makes sense to be held here.
  */
-abstract class BaseGitActivity : AppCompatActivity() {
+abstract class BaseGitActivity : ContinuationContainerActivity() {
```

Comment:
All non-commit operations run through `BaseGitActivity` so it made sense to simply have it extend our container